kafka-client/src/main/java/dev/responsive/kafka/api/config/ResponsiveConfig.java

```diff
@@ -246,6 +259,26 @@ public class ResponsiveConfig extends AbstractConfig {
           REMOTE_TABLE_CHECK_INTERVAL_MS_DEFAULT,
           Importance.LOW,
           REMOTE_TABLE_CHECK_INTERVAL_MS_DOC
+      ).define(
```

Comment:
I think it would be better to compute the default from the write consistency level. Don't want to block the PR on it though.